src/main/java/fr/adrienbrault/idea/symfony2plugin/translation/parser/TranslationStringParser.java

```diff
@@ -51,10 +51,10 @@ public TranslationStringMap parse(File file) {
 
         TranslationStringMap string_map = new TranslationStringMap();
 
-        Matcher matcher = Pattern.compile("'([\\w]+)'\\s=>.*?array\\s\\((.*?)\\s\\),", Pattern.MULTILINE).matcher(string);
+        Matcher matcher = Pattern.compile("'([\\w]+(\\+intl\\-icu)?)'\\s=>.*?array\\s\\((.*?)\\s\\),", Pattern.MULTILINE).matcher(string);
```

Comment:
I just noticed, that the TranslationStringParser isn't used anywhere, do I see this correctly? In that case my changes here do not have any effect and the classes can be removed?